docs/contributing/core/index.md

```diff
@@ -248,9 +248,17 @@ It takes 5-10 minutes to run the full unit test suite.
 If you run acceptance tests with the `--all` option, it will run tests in a real browser.
 This takes 30-40 minutes to run.
 This may repeatedly launch and close browser windows that gain focus, disrupting you from doing any other work.
-If this happens, you can install the `chromedriver` OS package.
-See https://developer.chrome.com/docs/chromedriver.
-Then run `export ROBOT_BROWSER="headlesschrome"` and again run `bin/test --all`.
+If this happens, you can use `headlesschrome` as testbrowser by setting an environment variable `export ROBOT_BROWSER="headlesschrome"` and again run `bin/test --all`.
+
+Since we use [Playwright](https://playwright.dev/) for our robottests you have to install it first.
+`plone.app.robotframework` provides a script for that:
```

Comment:
```suggestion
`plone.app.robotframework` provides a script to install Playwrite browsers.
```

Reply:
small typo: it should be `Playwright`

Reply:
Thanks, will fix.

Reply:
Fixed in https://github.com/plone/documentation/pull/1776/commits/3ac55229e9264b198eda284d8341028df8e89f01 in https://github.com/plone/documentation/pull/1776